src/types/id.ts

```diff
@@ -14,6 +14,7 @@ export type UserId = Branded<string, "UserId">;
 export type WorldId = Branded<string, "WorldId">;
 export type WorldEventId = Branded<string, "WorldEventId">;
 export type WorldSlug = Branded<string, "WorldSlug">;
+export type ElementId = Branded<string, "ElementId">;
```

Comment:
Please keep them in alphabetic order since the list might grow even larger

Reply:
done: https://github.com/sparkletown/sparkle/pull/2953/commits/a52527d7fce74b5f15447723260cc7715c8614b3